core/tests/test_utils.py

```diff
@@ -220,7 +272,13 @@ def get_expected_logout_url(self, slug):
         return current_user_services.create_logout_url(slug)
 
     def _parse_json_response(self, json_response, expect_errors=False):
-        """Convert a JSON server response to an object (such as a dict)."""
+        """Convert a JSON server response to an object (such as a dict).
+
+        Args:
+            json_response: dict (json). Contains json response from api call.
+            expect_errors: bool. If False, the json response is checked
```

Comment:
This is an implementation detail. Instead, it's better to describe what expect_errors represents. In this case: `Whether json_response is expected to represent an error response.`